addons/binding/org.openhab.binding.icloud/README.md

```diff
@@ -0,0 +1,77 @@
+# iCloud Binding
+
+The Apple [iCloud](https://www.apple.com/icloud/) is used to retrieve data such as the battery level or current location of one or multiple Apple devices connected to an iCloud account.
+Updates are quick and accurate without significant battery time impact.
+The Binding also offers access to the "Find My iPhone" function.
+
+An Apple account is required. Two factor authentication is supported.
+
+Please note: Application specific passwords are not supported.
+You´ll need to provide the account password to the binding.
+
+## Discovery
+
+An iCloud account can be added as a thing and needs to be configured with your ID and password.
+The devices registered to this account will then be automatically discovered.
+
+## Binding Configuration
+
+The Binding has no configuration options, all configuration is done at Thing level.
+
+## Thing Configuration
+
+The Binding provides two Thing types.
+
+### iCloud Account (Bridge) Thing
+
+The following table lists the configuration parameters:
+
+| Parameter            | Description |                             |
+|----------------------|-------------|-----------------------------|
+| appleId              |             | mandatory                   |
+| password             |             | mandatory                   |
+| refreshTimeInMinutes |             | optional, 5 minutes default |
```

Comment:
Do those three columns still make sense? Doesn't seem like it.

Please put all parameter names as well as channel IDs below into `code blocks`.

Reply:
Please describe the process of Update somewhere in a paragraph. Short draft:

> All Things are updated according to the configured refresh time of their shared account Bridge. You may force an update by sending the REFRESH command to any channel of the Bridge Things.